pl_examples/quantization/README.md

```diff
@@ -0,0 +1,86 @@
+Step-by-Step
```

Comment:
can we make sure the folder is called "intel_quantization" and something similar, not "quantization"?

Reply:
I would prefer not, this would be generally confusing to the user as we currently have 3 ways to perform quantization and this folder shouldn't contain them 3 ideally.

Reply:
I'd expect the word "quantization" in PyTorch first party repos to refer to PyTorch's native quantization solution.  Since this PR is proposing integrating a 3p tool with similar functionality, it would be good to be clear in the naming.

Reply:
Thanks for your comments @tchaton @vkuzo. WIP to enable INC (for Intel Neural Compressor) Callback to support various quantization approaches e.g., post-training static quantization, post-training dynamic quantization, quantization-aware training, etc. Regarding Intel quantization w/ INC vs. PyTorch native quantization, Intel quantization follows the native quantization API/flow to allow the users to migrate from PyTorch native to Intel quantization easily and provides additional value to maximize the productivity powered by automatic tuning.

Reply:
now the folder is called "quantization/intel_neural_compressor"